en/docs/choreo-cli/using-personal-access-tokens.md

```diff
@@ -0,0 +1,125 @@
+# Personal Access Tokens Guide for Choreo CLI
+
+## Introduction
+Personal Access Tokens (PATs) allow you to authenticate without using your primary credentials, adding an extra layer 
```

Comment:
Suggestion:
Personal access tokens (PATs) provide a secure method to authenticate with the Choreo CLI, allowing you to manage access without relying on primary credentials.